Extra comma on line 21 of the package.json that causes errors with npm.

**Describe the bug**
There's an extra comma on line 21 of the package.json that causes errors. 

 "strapi-plugin-transformer": "^2.1.1",

**To Reproduce**
Steps to reproduce the behavior:
1. Clone repo
2. "npm i" in terminal
3. Error - 
npm ERR! JSON.parse Failed to parse JSON data.
npm ERR! JSON.parse Note: package.json must be actual JSON, not just JavaScript.

**Expected behavior**
Should install packages. 

**Solution**
Remove the comma.

--- a/package.json
+++ b/package.json
@@ -18,7 +18,7 @@
     "@strapi/plugin-users-permissions": "4.6.0",
     "mysql": "2.18.1",
     "strapi-plugin-import-export-entries": "^1.18.0",
-    "strapi-plugin-transformer": "^2.1.1",
+    "strapi-plugin-transformer": "^2.1.1"
   },
   "author": {
     "name": "A Strapi developer"

